Sources/StreamChat/APIClient/APIClient.swift

```diff
@@ -16,10 +16,10 @@ class APIClient {
     let decoder: RequestDecoder
 
     /// Used for reobtaining tokens when they expire and API client receives token expiration error
-    let tokenRefresher: (@escaping () -> Void) -> Void
+    var tokenRefresher: ((@escaping () -> Void) -> Void)?
```

Comment:
why do we make these optional?

Reply:
The explaination is here: https://github.com/GetStream/stream-chat-swift/pull/2762#issuecomment-1703066227